### ISSUE TITLE
Remove icons from extensions blocks

### ISSUE THREAD

**Issue by True-Fantom** [ScratchAddons/ScratchAddons#5644]
### Why this would be helpful

This addon can significantly reduce the size of the displayed code and can help simulate the Scratch 2.0 interface.

### How this addon works

![Screenshot_183](https://user-images.githubusercontent.com/116278271/213433030-7eb2c597-0ff7-4002-aafd-12908f634b50.png)

### Additional context

Would be great if it supports turbowarp custom extensions.

**Comment by mxmou** [comment]
This probably shouldn't affect hardware extensions because the icon is the only way to differentiate some hardware blocks:
![image](https://user-images.githubusercontent.com/51849865/213438882-74d16ac9-da3c-4d4e-91ca-144fa9913c8c.png)

**Comment by True-Fantom** [comment]
@mxmou Yes, I think you're right

**Comment by Secret-chest** [comment]
> This probably shouldn't affect hardware extensions because the icon is the only way to differentiate some hardware blocks:
> ![image](https://user-images.githubusercontent.com/51849865/213438882-74d16ac9-da3c-4d4e-91ca-144fa9913c8c.png)

But make it smaller, and except Makye Makey.

**Comment by penguinmoose** [comment]
> This probably shouldn't affect hardware extensions because the icon is the only way to differentiate some hardware blocks: ![image](https://user-images.githubusercontent.com/51849865/213438882-74d16ac9-da3c-4d4e-91ca-144fa9913c8c.png)
Yes, but the icon should only show more than one hardware extension is used. This should be customizable.

Unless extensions have unique colors: #5656

**Comment by WorldLanguages** [comment]
@penguinmoose Keep in mind that it would be difficult to "only hide icons if colors are different".

**Comment by Secret-chest** [comment]
Are there any projects that use multiple hardware extensions?

**Comment by BroJac5246** [comment]
> Are there any projects that use multiple hardware extensions?

Theoretically there could be a case.

**Comment by mxmou** [comment]
> Are there any projects that use multiple hardware extensions?

Even if there's only one, it might not be obvious which one is used when viewing another user's project.
